setup.py

```diff
@@ -30,6 +30,7 @@
     "flake8>=5.0.0",
     "time_machine",
     "pyproj>=3.2.0",  # Pyproj is an optional, best-effort runtime dependency
+    "pystac"
```

Comment:
as mentioned above, this should be a general dependency, not just test dependency